stdlib/public/TensorFlow/Random.swift

```diff
@@ -171,3 +171,132 @@ public final class NormalDistribution<T : BinaryFloatingPoint>
     return mean + standardDeviation * normal01
   }
 }
+
+@_fixed_layout
+public final class BetaDistribution {
+  public let alpha: Float
+  public let beta: Float
+  private let uniformDistribution = UniformFloatingPointDistribution<Float>()
+
+  public init(alpha: Float = 0, beta: Float = 1) {
+    self.alpha = alpha
+    self.beta = beta
+  }
+
+  public func next<G: RandomNumberGenerator>(using rng: inout G) -> Float {
+    // Generate a sample using Cheng's sampling algorithm from:
+    // R. C. H. Cheng, "Generating beta variates with nonintegral shape 
+    // parameters.". Communications of the ACM, 21, 317-322, 1978.
+    let a = min(alpha, beta)
+    let b = max(alpha, beta)
+    if a > 1 {
+      return BetaDistribution.chengsAlgorithmBB(alpha, a, b, using: &rng)
+    } else {
+      return BetaDistribution.chengsAlgorithmBC(alpha, b, a, using: &rng)
+    }
+  }
+
+  /// Returns one sample from a Beta(alpha, beta) distribution using Cheng's BB 
+  /// algorithm, when both alpha and beta are greater than 1.
+  ///
+  /// - Parameters:
+  ///     - alpha: First Beta distribution shape parameter.
```

Comment:
Align `-` with "Parameters".

```suggestion
  ///   - alpha: First Beta distribution shape parameter.
```